android/java/org/chromium/chrome/browser/util/BraveConstants.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.util;
+
+public final class BraveConstants {
+    public static final int SITE_BANNER_REQUEST_CODE = 33;
+    public static final int VERIFY_WALLET_ACTIVITY_REQUEST_CODE = 34;
+    public static final int USER_WALLET_ACTIVITY_REQUEST_CODE = 35;
+    public static final int MONTHLY_CONTRIBUTION_REQUEST_CODE = 36;
+    public static final int DEFAULT_BROWSER_ROLE_REQUEST_CODE = 37;
```

Comment:
nit: would be better to add a comment to mention the primary class name of these constants to describe association for clarity and search-ability